index.html

```diff
@@ -1476,9 +1476,9 @@ <h3>Issuance Date</h3>
 A <a>credential</a> MUST have an <code>issuanceDate</code> <a>property</a>. The
 value of the <code>issuanceDate</code> <a>property</a> MUST be a string value of
 an [[!RFC3339]] combined date and time string representing the date and time
-the <a>credential</a> was issued. Note that this date represents the earliest
+the <a>credential</a> becomes valid. Note that this date represents the earliest
```

Comment:
Perhaps:
> the <a>credential</a> becomes valid, which could be a date and time in the future.

Then we don't need the redundant:
> Note that this represents . . .

Reply:
@brentzundel. The original text talked about both the VC and the credential subject. So I have kept this for now, unless people think it is unnecessary.

Reply:
@brentzundel . Take a look at the new Diff and see if you like it now

Reply:
I like it better. The different sentence constructions help to better clarify that they are talking about two different things. 
I wonder if the sentence starting with "Note" should actually be a note, set apart as such in the text.